Source/Core/Core/PowerPC/JitArm64/JitArm64_LoadStore.cpp

```diff
@@ -274,6 +274,19 @@ void JitArm64::SafeStoreFromReg(s32 dest, u32 value, s32 regOffset, u32 flags, s
   gpr.Unlock(ARM64Reg::W0, ARM64Reg::W1, ARM64Reg::W30);
 }
 
+FixupBranch JitArm64::BATAddressLookup(ARM64Reg addr_out, ARM64Reg addr_in, ARM64Reg tmp)
+{
+  tmp = EncodeRegTo64(tmp);
+
+  MOVP2R(tmp, PowerPC::dbat_table.data());
```

Comment:
Do you think it is worth to place the BAT table somewhere within the PPC_State?

Reply:
In that case we would need to have an addition instruction in here instead of the MOVP2R (since we wouldn't be putting the BAT table at the very beginning of PPCState). So we'd save maybe one instruction (if MOVP2R needs two instructions), which isn't all that much.